### ISSUE TITLE
Configurable 'old' and 'new' day rendering

### ISSUE THREAD

**Issue by loginet-vfeher** [year-calendar/js-year-calendar#14]
The library simply ignore those days (append old & new css classess), which do not belong to a given month during rendering. On the screen Jan 1st is on Tuesday which rendered correctly, but Monday as Dec 31st simply dropped.

Actual result:
![image](https://user-images.githubusercontent.com/35222014/65150662-e2c8d080-da24-11e9-8dda-123062885631.png)

We would appreciate that if the library could accept such a config parameter, for the full month rendering, as represented below.

Desired result:
![image](https://user-images.githubusercontent.com/35222014/65150733-fe33db80-da24-11e9-9038-431a0e5cc2e5.png)

Note that, weekstart is irrelevent actually.

**Comment by Paul-DS** [comment]
I don't plan to implement this feature for now, given that the calendar is supposed to display all months in a year, the 'old' and 'new' days are displayed on the previous/next month.

I keep this issue open, if other people need this feature, don't hesitate to let a comment here, I'll think about it if many people request it.

**Comment by wobility** [comment]
Hi, 

Yes I need this feature too. 👍🏼

**Comment by sachin1989solanki** [comment]
Yes I need to implement this feature, Please provide us as soon as possible.
